### PR TITLE
update Alpine and Golang

### PR DESCRIPTION
**Pull request by PascalBourdier**
use Go 1.17 and Alpine 3.15

### CODE REVIEW

**Comment by tianon** [comment]
I agree with making these updates, but similar to https://github.com/tianon/gosu/pull/101#issuecomment-1062122614 (see also #98), I want to make sure I'm not merging this with a false expectation -- I do not plan to make a new release with these updates unless there's a more compelling reason to do so than "my security scanner says it's vulnerable" (because the scan result itself is an overzealous scanner -- ultimately the vulnerability is a false positive).

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,11 @@
 module github.com/tianon/gosu
 
-go 1.16
+go 1.17
+
+require github.com/opencontainers/runc v1.1.0
 
 require (
```
Comment (tianon) [inline go.mod:7]: This split is interesting -- is this something the `go` tooling did automatically?

**Comment by PascalBourdier** [comment]
> Would you also be willing to update `hub/Dockerfile.alpine`? 🙏
> 
> (I'm happy to take over from here if you'd prefer!)

I fixed it

File: go.mod (hunk 1 above)
Comment (PascalBourdier) [reply go.mod:7]: it seems native in Golang 1.17 : https://go.dev/doc/go1.17#tools

**Comment by tianon** [comment]
Thanks!

**Comment by slakwa** [comment]
@tianon I have a question about this change. Shouldn't also the version of gosu be updated from 1.14 to 1.15 for other tools using this library to be able to get this change?

**Review by Yedfree — APPROVED** [review]

**Pull request merged**